server/src/testFixtures/java/io/crate/operation/aggregation/AggregationTestCase.java

```diff
@@ -201,9 +226,11 @@ public Object executeAggregation(Signature maybeUnboundSignature,
             terminatePartialAggFunction = aggregationFunction;
         }
 
-        Version minNodeVersion = randomBoolean()
-            ? Version.CURRENT
-            : Version.V_4_0_9;
+        Version minNodeVersion = Version.CURRENT;
+        if (randomMinNodeVersion && randomBoolean()) {
+            minNodeVersion = Version.V_4_0_9;
```

Comment:
We could get rid of this as we do not support < 5.x on master. Or we change it to some 5.x version if it is still used.